Emphasize Ingest over POST, SDKs, and open source (#172)
Unstructured-IO/docs: api-reference/api-services/free-api.mdx

## Quickstart

> These examples use your local machine. They preprocess source (input) files on your local machine, and the Unstructured Serverless API delivers the processed data to a destination (output) location, also on your local machine. Data is processed on Unstructured-hosted compute resources.

Reviewer: "They preprocess source (input) files on your local machine" this makes it sound as if the data was processed on the local machine. 
It is ingested from the local machine, but it is processed on Unstructured-hosted compute resources as you state later. But the paragraph is a bit confusing.

> These examples use your local machine. They preprocess source (input) files on your local machine, and the Unstructured Serverless API delivers the processed data to a destination (output) location, also on your local machine. Data is processed on Unstructured-hosted compute resources.

Author: ```suggestion
These examples use your local machine. They send source (input) files from your local machine to the Unstructured Serverless API which delivers the processed data to a destination (output) location, also on your local machine. Data is processed on Unstructured-hosted compute resources.
```
